src/nomad_simulation_parsers/schema_packages/h5md.py

```diff
@@ -16,25 +16,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# TODO: test_h5md_parser.py::test_md currently fails with jmespath parsing errors
+# when mapping HDF5 attributes that use the @ symbol (e.g., "@version", "@name").
```

Comment:
@ladinesa the annotations for attributes in the hdf5 are no longer working for me, e.g., l 128 .@version

Reply:
Can you try something like this? `'.i[?"@name"==\'version\']`, (see vasp.py)

Reply:
I don't really understand which part of this you are suggesting to use. The issue is I am trying to access 
```
{
    'h5md': {
        '@version': [1, 0]   
    }
}
```
but for `add_mapping_annotation(general.Program.version, HDF5_KEY, '."@version"')`  mapping parser strips the `" "`, leaving only `@version` so I get a failed jmespath

I tried escaping the inner quotes in some different ways:

`'."@version"' `
`".\'@version\'" `
`'."@version"' `
`'.\'@version\'' `

Reply:
sorry misunderstood it you can escape it with `'\"@version\"'`

Reply:
If you have it as literal you have to remove the slashes

Reply:
```
from nomad.parsing.file_parser.mapping_parser import Path
p = Path(path=r'"@version"')
p.get_data({'@version': 1})
1
```